### PR TITLE
PE-342: Move commands declaration to its own place

### PR DESCRIPTION
**Pull request by matibat**
Cleans the `src/index.ts` by moving the command declarations to multiple separate files under `/src/commands` and creating `utils.ts`

### CODE REVIEW

**Comment by fedellen** [comment]
Just self tested this: 

`get-mempool` and `get-balance` both return:

```shell
(node:4727) UnhandledPromiseRejectionWarning: Error: No such parameter parentFolderId
    at Function.get (/Users/derek/ardrive/ardrive-cli/lib/CLICommand/parameter.js:38:19)
    at new Parameter (/Users/derek/ardrive/ardrive-cli/lib/CLICommand/parameter.js:7:40)
    at /Users/derek/ardrive/ardrive-cli/lib/CLICommand/cli_command.js:21:27
    at Array.forEach (<anonymous>)
    at setCommanderCommand (/Users/derek/ardrive/ardrive-cli/lib/CLICommand/cli_command.js:20:34)
    at CLICommand.setCommand (/Users/derek/ardrive/ardrive-cli/lib/CLICommand/cli_command.js:75:9)
    at new CLICommand (/Users/derek/ardrive/ardrive-cli/lib/CLICommand/cli_command.js:52:14)
    at Object.<anonymous> (/Users/derek/ardrive/ardrive-cli/lib/commands/upload_file.js:18:1)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:4727) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:4727) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by matibat** [comment]
Thanks for finding that @fedellen, some parameters weren't defined there. It's now fixed at 1d26a5ea8a0791010b3ae5b52d7b8308efb52a03

**Comment by matibat** [comment]
`get-mempool` still fails...

**Comment by matibat** [comment]
Fixed two wrong imports. That fixes the command failures for `get-balance` and `get-mempool`

File: src/parameter_declarations.ts [hunk 1]
```diff
@@ -52,7 +62,77 @@ Parameter.declare({
...
+
+Parameter.declare({
+	name: ConfirmationsParameter,
+	aliases: ['-c', '--confirmations'],
+	description: 'Number of confirmations to determine if a transaction is mined'
+});
+
+Parameter.declare({
+	name: ParentFolderIdParameter,
+	aliases: ['--parent-folder-id'],
```
Comment (fedellen) [inline src/parameter_declarations.ts:106]: This should have -f as a short option. The ones below are missing their short options from below as well: 

```ts
// old index.ts

.requiredOption(
	'-f, --parent-folder-id <parent folder id>',
	`the ArFS folder ID for the folder in which this file will reside (i.e. its parent folder)
	• To upload the file to the root of a drive, use the root folder ID of the drive`
)
.option(
	'-l, --local-file-path <local file path>',
	`the path on the local filesystem for the file that will be uploaded`
)
.option(
	'-d, --dest-file-name <destination file name>',
	`(OPTIONAL) a destination file name to use when uploaded to ArDrive`
)
```

Comment (matibat) [reply src/parameter_declarations.ts]: Yep! I missed some options. Added at 96584cc700f98fa16c9b26cb898cd5297063ac33

**Review by fedellen — APPROVED** [review]
Looking good on my end!

**Pull request merged**